### ISSUE TITLE
SC2089 Multiple backslash with quotes showing warning

### ISSUE THREAD

**Issue by arafatx** [koalaman/shellcheck#2638]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): 
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

Just want to print this simple statement
```sh

#!/bin/bash

var=", \\\"ttl_sec\\\": 30"

echo "${var}"

```

The output is

`, \"ttl_sec\": 30`

#### Here's what shellcheck currently says:

Quotes/backslashes will be treated literally. Use an array.
See SC2089.

#### Here's what I wanted or expected to see: 
Improve the suggestion by shellcheck. Is there anything wrong with the multiple backslashes? Thank you.

**Comment by arafatx** [comment]
Sorry, does not seems to have issue on the shellcheck site. Probably I'm using the old shellcheck
